### PR TITLE
Use Open JDK and specify v8

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

File: docs/source/getting-started/installation.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ The SDK currently runs on MacOS or Linux. For Windows support, we recommend inst
 You need to install:
 
 1. `Visual Studio Code <https://code.visualstudio.com/download>`_.
-2. `JDK 8 or greater <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_.
+2. `JDK 8 <https://openjdk.java.net/install/>`_.
```
Comment (hurryabit) [inline docs/source/getting-started/installation.rst]: This page does not contain installation instructions for Mac. :(

I followed the instructions at https://github.com/AdoptOpenJDK/homebrew-openjdk#readme but I'm not sure we want to tell this to our users.

Comment (bethaitman) [reply docs/source/getting-started/installation.rst]: ugh, okay, this is going to need some more thought, isn't it...

**Comment by bitonic** [comment]
Note bazel already uses OpenJDK -- or in another words we're already testing every day with OpenJDK. We should really try to instruct people to use OpenJDK, too.

**Comment by dasormeter** [comment]
> Note bazel already uses OpenJDK -- or in another words we're already testing every day with OpenJDK. We should really try to instruct people to use OpenJDK, too.

@francesco-da @aherrmann-da @majcherm-da what do we think about recommending Zulu for all OSes, and using this in devenv as well? Zulu looks to have pretty well documented and easier to find OpenJDK for all the main OSes -- otherwise you need to stitch together different versions with odd combos from three places like i have on this PR now

**Comment by majcherm-da** [comment]
These are all OpenJDKs just built by different companies and you're right that Zulu/Azul seems to be the most user friendly - you can easily find all different versions of specific releases, so I'm for it.

**Comment by dasormeter** [comment]
@moritzkiefer-da / @bethaitman / @francesco-da : are we about to imminently update this to recommend JDK 11 for a monday/tuesday release version of SDK?

**Comment by cocreature** [comment]
We will upgrade once I get the OK from the consumers of our artifacts. If that happens Monday/Tuesday I’ll happily merge the upgrade and do a release but I expect that it will probably take a bit longer for that to happen.

**Comment by bitonic** [comment]
@dasormeter we should go ahead and update the instructions with OpenJDK 8, for now. we'll update again when OpenJDK 11 comes.

**Comment by bitonic** [comment]
@dasormeter  personally I'd prefer an "official" link per OS rather than a single link to some third party packaging for all OS's, but this is obviously very much down to personal preference. I generally don't like / trust third parties repackaging openly available software.

**Comment by dasormeter** [comment]
this got confused in merges -- closing and replacing with #1877

**Pull request closed without merging**